Documentation/recommended-labels.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Recommended Labels
```

Comment:
This doc feels more like a design doc than something that needs to be in the user documentation. User docs are really focused around what is needed for configuration and troubleshooting. This doc feels more like a description of this feature that was implemented, which is more the purpose of design docs. How about if we move this to be a design doc (under `/design/ceph/`) instead of under `/Documentation? If users want to know what labels are available, they can just inspect them at runtime.